python/numpydoc.rst

```diff
@@ -706,6 +720,14 @@ If the keys are well-known, document the keys using a pattern similar to :ref:`p
 
 Notice how the keys are shown by enclosing the quoted strings in double backticks to clarify that the keys are `str` types.
 
+Mappings of complex types can potentially have very long type descriptions.
```

Comment:
I guess the duplication is okay since people will potentially be coming at this from different directions.